<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>desWidgetClass</class>
 <widget class="QMainWindow" name="desWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>desWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>601</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Dé</string>
     </attribute>
     <widget class="QPushButton" name="afficheDe">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher la valeur</string>
      </property>
     </widget>
     <widget class="QPushButton" name="boutonLancerDe">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tirer le dé</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelResultat">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="scoreTot">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Score total</string>
      </property>
     </widget>
     <widget class="QLabel" name="afficheScore">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>220</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="remiseZero">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>160</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset le score</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Déhisto</string>
     </attribute>
     <widget class="QPushButton" name="tirerDehisto">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Tirer le de</string>
      </property>
     </widget>
     <widget class="QPushButton" name="afficherDehisto">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher de</string>
      </property>
     </widget>
     <widget class="QLabel" name="afficheDehisto">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>111</width>
        <height>301</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>7</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
     </widget>
     <widget class="QPushButton" name="tirerNDehisto">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>217</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>nombre de lancers :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nbrJets">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actiondes">
   <property name="text">
    <string>des</string>
   </property>
  </action>
  <action name="actiondehisto">
   <property name="text">
    <string>dehisto</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="desWidget.qrc"/>
 </resources>
 <connections/>
</ui>
